queries.sql

```diff
@@ -23,3 +23,71 @@ WHERE name != 'Gabumon';
 
 SELECT * FROM animals
 WHERE weight_kg >= 10.4 AND weight_kg <= 17.3;
+
```

Comment:
Great job @jkanyi-web 🤝

- [ ] As stated in the project requirements, kindly set the species column to unspecified, then roll back the change. You may also verify that the change was made and rolled back.
![transaction-1](https://github.com/jkanyi-web/Vet-Clinic-DB/assets/63917725/69d7da72-2569-467e-9c23-c4f620028e28)